wiki/Skinning/Compendium/Minimalistic_&_Flat/en.md

```diff
@@ -0,0 +1,198 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Minimalistic and Flat
+Undefined Translation
```

Comment:
is this a placeholder?

Reply:
no its not. Something must have gone wrong in my programm. Probably forgot the translation